src/cmd-buildextend-gcp

```diff
@@ -111,7 +112,15 @@ def run_ore():
     checksum = sha256sum_file(build_tarball)
     size = os.path.getsize(build_tarball)
 
-    buildmeta['gcp'] = {'image': f"{base_name}-{args.build.replace('.', '-')}"}
+    gcp_name = f"{base_name}-{args.build.replace('.', '-')}"
+    url_path = urllib.parse.quote((
```

Comment:
Hmm sorry for making you redo this...We shouldn't actually quote the `/` components, I more meant the values.  It doesn't matter for us since there's no security boundary here but if you're writing a web app, if someone happens to put `/` in a field, you want that to be quoted (for robustness and also in some cases it can lead to unexpected argument injection).

Reply:
Agreed on avoiding subsystem injection/XSS when allowing for untrusted input. I _think_ in this case it the quoting, even on known input, should be fine. EG:

```
In [1]: import urllib.parse                                         

In [2]: urllib.parse.quote('/this/is-a/test.of.the.quotes')         
Out[2]: '/this/is-a/test.of.the.quotes'
```

If `quote_plus` was in use, it would be problematic.

Reply:
The signature for `urllib.parse.quote` defaults `/` as safe, by default. See https://docs.python.org/3/library/urllib.parse.html#url-quoting.

Reply:
Ah right, cool.